app/src/test/java/io/gnosis/safe/ui/transactions/details/TransactionDetailsViewModelTest.kt

```diff
@@ -108,6 +121,7 @@ class TransactionDetailsViewModelTest {
                 any()
             )
         } returns transactionDetails
+        coEvery { transactionLocalRepository.updateLocalTx(any(), any<String>()) } returns null
```

Comment:
I would expect in the test that we check that this method was called. This is a crucial step and easy to verify via the  mocks